fairlearn/reductions/_moments/utility_parity.py

```diff
@@ -3,16 +3,30 @@
 
 import pandas as pd
 import numpy as np
+
 from .moment import ClassificationMoment
 from .moment import _GROUP_ID, _LABEL, _PREDICTION, _ALL, _EVENT, _SIGN
-from fairlearn._input_validation import _MESSAGE_RATIO_NOT_IN_RANGE
+from fairlearn._input_validation import _MESSAGE_RATIO_NOT_IN_RANGE, _validate_and_reformat_input
 from .error_rate import ErrorRate
 
+
 _UPPER_BOUND_DIFF = "upper_bound_diff"
 _LOWER_BOUND_DIFF = "lower_bound_diff"
 _MESSAGE_INVALID_BOUNDS = "Only one of difference_bound and ratio_bound can be used."
 _DEFAULT_DIFFERENCE_BOUND = 0.01
 
+_CTRL_EVENT_FORMAT = "control={0},{1}"
```

Comment:
What if either one of them already has a comma? How do we distinguish between control and event? Or is that something we don't care about?

Reply:
I could make it `event={1}` too, I suppose. But so long as this generates unique strings, I don't believe we ever care.

Reply:
Actually, things we have generally set the event to be `Label=<STRING>`